### ISSUE TITLE
ValueError: Unknown field auth_status.

### ISSUE THREAD

**Issue by SharkWipf** [thebigmunch/google-music-scripts#7]
- `gms -v` : `google-music-scripts 2.0.0`.
- Installed through `pip3.6 install --user google-music-scripts`.

Running `gms` followed by any of the commands results in `gms` asking for the authentication key (on first run) or saying "Logging in" (on subsequent runs), but then, after ~20 seconds, failing with the following traceback: 
```
[2018-11-29 07:15:26] Logging in to Google Music
Traceback (most recent call last):
  File "/home/sebastiaan/.local/bin/gms", line 11, in <module>
    sys.exit(gms())
  File "/home/sebastiaan/.local/lib64/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/sebastiaan/.local/lib64/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/sebastiaan/.local/lib64/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/sebastiaan/.local/lib64/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/sebastiaan/.local/lib64/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/sebastiaan/.local/lib64/python3.6/site-packages/google_music_scripts/commands/upload.py", line 117, in upload
    mm = google_music.musicmanager(username, uploader_id=uploader_id)
  File "/home/sebastiaan/.local/lib64/python3.6/site-packages/google_music/api.py", line 51, in musicmanager
    token=token
  File "/home/sebastiaan/.local/lib64/python3.6/site-packages/google_music/clients/musicmanager.py", line 64, in __init__
    self._upauth(uploader_id, uploader_name)
  File "/home/sebastiaan/.local/lib64/python3.6/site-packages/google_music/clients/musicmanager.py", line 70, in _upauth
    self._call(mm_calls.UpAuth, uploader_id, uploader_name)
  File "/home/sebastiaan/.local/lib64/python3.6/site-packages/tenacity/__init__.py", line 292, in wrapped_f
    return self.call(f, *args, **kw)
  File "/home/sebastiaan/.local/lib64/python3.6/site-packages/tenacity/__init__.py", line 358, in call
    do = self.iter(retry_state=retry_state)
  File "/home/sebastiaan/.local/lib64/python3.6/site-packages/tenacity/__init__.py", line 331, in iter
    raise retry_exc.reraise()
  File "/home/sebastiaan/.local/lib64/python3.6/site-packages/tenacity/__init__.py", line 167, in reraise
    raise self.last_attempt.result()
  File "/usr/lib64/python3.6/concurrent/futures/_base.py", line 425, in result
    return self.__get_result()
  File "/usr/lib64/python3.6/concurrent/futures/_base.py", line 384, in __get_result
    raise self._exception
  File "/home/sebastiaan/.local/lib64/python3.6/site-packages/tenacity/__init__.py", line 361, in call
    result = fn(*args, **kwargs)
  File "/home/sebastiaan/.local/lib64/python3.6/site-packages/google_music/clients/base.py", line 77, in _call
    return call.parse_response(response.headers, response.content)
  File "/home/sebastiaan/.local/lib64/python3.6/site-packages/google_music_proto/musicmanager/models.py", line 41, in parse_response
    if not self.check_success(res_body):
  File "/home/sebastiaan/.local/lib64/python3.6/site-packages/google_music_proto/musicmanager/calls.py", line 495, in check_success
    return response.HasField('auth_status') and response.auth_status == upload_pb2.UploadResponse.OK
ValueError: Unknown field auth_status.
```

**Comment by thebigmunch** [comment]
Forgot to release a bugfix release of ``google-music-proto`` ^_^

Running ``pip install -U google-music-scripts`` or ``pip install google-music-proto==2.1.1`` should get you the correct version of ``google-music-proto``.

That being said, I only saw this issue when the call is failing for some other reason. The new version should print out that reason in an exception. I noticed it when I decided to switch the account I was testing on, and Google said I had released the max accounts per machine limit (2) despite removing the device from the previous account. I ended up just setting ``uploader-id`` for the new test account in the new [config file](https://google-music-scripts.readthedocs.io/en/latest/#configuration) to that of a different machine I have. I may make a way around this in the future. Just mentioning all this in case that's what's happening here.
